packages/app/src/vis-packs/nexus/utils.ts

```diff
@@ -48,7 +48,7 @@ export function isNxGroup(entity: Entity): boolean {
     isGroup(entity) &&
     (isNxDataGroup(entity) ||
       hasAttribute(entity, 'default') ||
-      hasAttribute(entity, 'NX_class'))
+      getAttributeValue(entity, 'NX_class') === 'NXentry')
```

Comment:
This would remove the support for NXroot without `default` as pointed out in https://github.com/silx-kit/h5web/pull/833#discussion_r741092093